projects/js-packages/publicize-components/src/components/one-click-sharing-modal/index.js

```diff
@@ -0,0 +1,57 @@
+import { Text, ThemeProvider, useBreakpointMatch } from '@automattic/jetpack-components';
+import { Modal, ResponsiveWrapper } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { ShareButtons } from '../share-buttons/share-buttons';
+import background from './background.svg';
+import illustration from './illustration.svg';
+import styles from './styles.module.scss';
+
+const OneClickSharingModal = ( { onClose } ) => {
+	const [ isSm ] = useBreakpointMatch( 'sm' );
+
+	return (
+		<Modal className={ styles.modal } onRequestClose={ onClose }>
+			<ThemeProvider>
+				<div className={ styles.container }>
+					<img className={ styles.background } src={ background } alt="" />
+
+					<div className={ styles.column }>
+						<Text className={ styles.title }>
+							{ __( 'Introducing Simplified Sharing ', 'jetpack' ) }
+						</Text>
+						<Text className={ styles.description } variant="body-small">
+							{ __(
+								'Hey there! Quick update for you. You can now effortlessly share your new posts on Twitter, WhatsApp, or any other platform.',
```

Comment:
This explicitly says Twitter for the discoverability of semi-supporting Twitter again